LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,256 +1,83 @@
-
-    (b) rightfully received from a third Party not in breach of any obligation of
-        confidentiality;
-
-    (c) independently developed by a Party without access to Confidential
-        Information of the other Party;
-
-    (d) known to the other Party at the time of disclosure;
-
-    (e) produced in compliance with applicable law or a court order, provided the
-        other Party is given notice and opportunity to intervene; or
-
-    (f) it does not constitute a trade secret and more than five (5) years have
-        elapsed from the date of disclosure. In addition to any other rights or
-        remedies available, each Party shall be entitled to enforcement of such
-        obligations by court injunction.
-
-23. This Agreement shall be construed, interpreted, and governed by the laws of
-    the Federal Republic of Germany.
-
-24. You may assign this Agreement, in whole or in part (whether by operation of
-    law or otherwise), with prior consent from us, which shall not be
-    unreasonably withheld or delayed. We may assign this Agreement or any of its
-    rights hereunder or delegate any of its obligations hereunder with prior
-    notice to you. Any attempt to assign this Agreement other than in accordance
-    with this provision shall be null and void.
-
-25. This Agreement constitutes the complete agreement between the Parties and
-    supersedes all prior or contemporaneous discussions, representations, and
-    proposals, written or oral, with respect to the subject matters discussed
-    herein. No modification of this Agreement will be effective unless contained
-    in a written document executed by an authorized representative of each
-    Party. If any provision of the Agreement is found void or unenforceable, the
-    remainder will remain valid and enforceable according to its terms. If any
-    remedy provided is determined to have failed for its essential purpose, all
-    limitations of liability and exclusions of damages set forth in this
-    Agreement shall remain in effect.
+The grant of rights in section 1 are conditional, provided that You do all of the following:
+
+(a) You have purchased an appropriate **Pricing Plan** (Startup & Individual / Small Enterprise / Enterprise) and the required amount of seats to cover all individual users of the Software. For clarity, each individual user is counted as one seat.
+
+(b) In the case that You are distributing the Software as as part of an Embedded System, You have purchased an appropriate quantity of **Royalties**, one Royalty for each Embedded System.
+
+(c) You have fully paid all applicable fees (**Fees**) pursuant to Section 4.
+
+## 3. Limitations
+
+The License does not permit to distribute or make the Software publicly available alone and without integration into an Application or into an Embedded System. For this purpose you may use the Software under the GNU General Public License, version 3.
```

Comment:
```suggestion
The License does not permit to distribute or make the Software publicly available alone and without integration into an Application or into an Embedded System. For this purpose you may use the Software under the terms of the GNU General Public License, version 3.
```

Reply:
Copied this from the Royalty-free license as per the wording suggested by legal counsel.